src/core/login/create.ts

```diff
@@ -167,6 +167,6 @@ export async function createLogin(
   await loginFetch(ai, 'POST', kit.serverPath, request)
 
   kit.stash.lastLogin = now
-  await saveStash(ai, kit.stash)
-  return kit.login
+  await saveStash(ai, kit.stash as LoginStash)
+  return kit.login as LoginTree
```

Comment:
This doesn't make sense to me. How are you able to make this assertion?

Reply:
We are getting rid of the `Partial<>`, since we happen to know that these have all the stuff that should be there. Sorry, it's old code from before we had good type-checking, so things are a little looser here than they should be.